constraint/pkg/regorewriter/regorewriter_test.go

```diff
@@ -495,3 +506,369 @@ func TestRegoRewriterErrorCases(t *testing.T) {
+		name    string
+		path    string
+		wantErr error
+	}{
+		{
+			name:    "path not exist",
+			path:    "foo/bar",
+			wantErr: ErrReadingFile,
+		},
+		{
+			name: "path with test folder",
+			path: ".",
+		},
+		{
+			name: "path without test",
+			path: "../../",
+		},
+	}
+	for _, tc := range tests {
+		t.Run(tc.name, func(t *testing.T) {
+			rr, err := MockRegoWriter()
+			if err != nil {
+				t.Fatalf("Failed to create RegoRewriter %q", err)
+			}
+			if err := rr.AddLibFromFs(tc.path); !errors.Is(err, tc.wantErr) {
+				t.Errorf("AddLibFromFs() error = %v, wantErr %v", err, tc.wantErr)
+			}
+			if err := rr.AddBaseFromFs(tc.path); !errors.Is(err, tc.wantErr) {
+				t.Errorf("AddBaseFromFs() error = %v, wantErr %v", err, tc.wantErr)
+			}
+		})
+	}
+}
+
+func TestRegoRewriteEntryPoint(t *testing.T) {
+	tests := []struct {
+		name       string
+		prefix     string
+		content    string
+		wantResult map[string]string
+		wantError  error
+	}{
+		{
+			name:   "entry point imports lib",
+			prefix: "prefix.path",
+			content: `package lib.alpha
+import data.lib.alpha
+violation[{"msg":msg}] {
+	x := data.lib.alpha
+	y := data.lib.alpha.ext[_]
+	z := "data.lib.alpha"
+	data.lib.alpha.check[input.name]
+	ex := data.ext.alpha
+}`,
+			wantResult: map[string]string{"path": `package lib.alpha
+
+import data.prefix.path.lib.alpha
+
+violation[{"msg": msg}] {
+	x := data.prefix.path.lib.alpha
+	y := data.prefix.path.lib.alpha.ext[_]
+	z := "data.lib.alpha"
+	data.prefix.path.lib.alpha.check[input.name]
+	ex := data.ext.alpha
+}
+`},
+		}, {
+			name:   "entry point binds data.lib to var",
+			prefix: "prefix",
+			content: `package templates.stuff.MyTemplateV1
+import data.lib.alpha
+violation[{"msg":msg}] {
+	x := data.lib
+	y := x[_]
+}`,
+			wantResult: map[string]string{"path": `package templates.stuff.MyTemplateV1
+
+import data.prefix.lib.alpha
+
+violation[{"msg": msg}] {
+	x := data.prefix.lib
+	y := x[_]
+}
+`},
+		}, {
+			name:   "entry point uses data.lib[_]",
+			prefix: "prefix",
+			content: `package templates.stuff.MyTemplateV1
+import data.lib.alpha
+violation[{"msg":msg}] {
+	x := data.lib[_]
+}`,
+			wantResult: map[string]string{"path": `package templates.stuff.MyTemplateV1
+
+import data.prefix.lib.alpha
+
+violation[{"msg": msg}] {
+	x := data.prefix.lib[_]
+}
+`},
+		}, {
+			name:   "entry point references input",
+			prefix: "prefix",
+			content: `package templates.stuff.MyTemplateV1
+violation[{"msg":msg}] {
+  bucket := input.asset.bucket
+}`,
+			wantResult: map[string]string{"path": `package templates.stuff.MyTemplateV1
+
+violation[{"msg": msg}] {
+	bucket := input.asset.bucket
+}
+`},
+		},
+	}
+	for _, tc := range tests {
+		t.Run(tc.name, func(t *testing.T) {
+			rr, err := New(NewPackagePrefixer(tc.prefix), []string{"data.lib"}, []string{"data.ext"})
+			if err != nil {
+				t.Fatalf("Failed to create RegoRewriter %s", err)
+			}
+			if err := rr.AddEntryPoint("path", tc.content); err != nil {
```

Comment:
No tests expect AddEntryPoint to fail, so we can remove the `errors.Is` check and can just immediately `t.Fatalf`

Reply:
replaced with 
```
if err != nil {
    t.Fatalf
}
```